### ISSUE TITLE
Rewrap bug in Guiguts 1.3.1 Beta

### ISSUE THREAD

**Issue by charliehoward4dp** [DistributedProofreaders/guiguts#728]
Indent specifications, e.g., \/\*[40], worked in previous versions of GG (1.0.25 through 1.2.4) 
when they were nested within block quotes, but are not working in GG 1.3.1. See attach file and look for \/\*[30] and \/\*[40]. The former occurs once; the latter, twice.

What happens is that the no-wrap markers are treated as wrappable text, not as markers.

[rewrapbug.zip](https://github.com/DistributedProofreaders/guiguts/files/6367670/rewrapbug.zip)

**Comment by windymilla** [comment]
When I rewrap a small file containing the text below (including blank line start and end) in release 1.0.25, it treats the /*[30] as wrappable text if I use Rewrap All. However if I just select from the /# to the #/ and rewrap selection, it appears to work. Can you confirm that it does something similar for you?
```

/#
/*[30]
Cambridge, 14th February, 1871.
*/

When I came here last Friday I found everyone talking
#/

```

**Comment by charliehoward4dp** [comment]
I just re-tested this using the full-size file in "rewrapbug.zip", using GG 1.0.25 and GG 1.2.4.  "Rewrap All" worked properly with both of them (two total tests, one with each version of GG).

When using and selecting just the fragment in your question above, with a blank line before and a blank line  after (but neither of those blank lines included in the selection), "Rewrap Selection" worked properly in 1.0.25 and in 1.2.4, and both gave the result below both ways.  Using the same text with "Rewrap All" worked properly with both versions as well. (This was 4 total tests of the fragment):


/#

/*[30]
                              Cambridge, 14th February, 1871.
*/

  When I came here last Friday I found everyone talking
#/

(Notice that all six tests added a blank line between the opening /# and the opening /*, which had been on adjacent lines before the rewrap.)

Maybe the ancient "fix" that was done before 1.0.24 was to pre-scan for consecutive opening markers and separate them with a blank line?
